modules/core/src/main/java/org/apache/synapse/message/store/impl/jms/JmsStore.java

```diff
@@ -49,6 +49,8 @@
 import java.util.NoSuchElementException;
 import java.util.Properties;
 import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class JmsStore extends AbstractMessageStore {
```

Comment:
Are we doing this only for JmsStore? What about other message stores?

Reply:
At the moment we did this only for JMS Store.

Reply:
Let's rename the PR title as JMS Message Store.

Reply:
Done